MultiscaleFusionFilters/ITKPhaseCorrelationImageRegistration.cpp

```diff
@@ -0,0 +1,244 @@
+// File automatically generated
```

Comment:
Should this be removed now? Or is this file occasionally automatically regenerated?

Reply:
The file is regenerated automatically. This is helpful if the script that generates the files is updated for example, like in this PR.

Reply:
Yes.

Reply:
See #3